documentation/modules/proc-jmxtrans-deployment.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+// assembly-jmxtrans.adoc.
+
+[id='proc-jmxtrans-deployment-{context}']
+= Configuring a JMXTrans deployment
+
+You can configure a JmxTrans deployment using the jmxTrans property in the following resources:
+
+* `Kafka.spec.kafka.jmxOptions`
+
+A JmxTrans deployment can read from a secure or insecure Kafka broker.
+
+To configure a JmxTrans deployment, define the following properties:
+
+* `Kafka.spec.kafka.jmxOptions.jmxTrans.outputDefinitions`
+* `Kafka.spec.kafka.jmxOptions.jmxTrans.queries`
```

Comment:
Link down to the appendix?

Reply:
Something like: 

For more information on these properties see, xref:type-JmxTransSpec-reference[`JmxTransSpec` schema reference].